### DISCUSSION TITLE
Latest commit date and keywords for Repo Extra Pins

### DISCUSSION THREAD

**Discussion by ToNyRANDRIAMANANTSOA** [anuraghazra/github-readme-stats#3923] [Ideas]
Hi, 
Could we include a date of the lastest commit/update with the [Extra Pins](https://github.com/anuraghazra/github-readme-stats#github-extra-pins) for repositories? And another option for the keywords ?

Something like the `show_user` option: `show_last_commit_date` or `show_keywords` ?

Something like the github search result :
![image](https://github.com/user-attachments/assets/9118ed6d-e3ff-4626-923f-8bd0f2f05abf)

Currently, what we can have at most is like:
![image](https://github.com/user-attachments/assets/640a2358-931b-4b11-9b17-7c563ad17508)

**Comment by hesreallyhim** [comment]
i can add this to my "repo-focused" fork...

**Reply by hesreallyhim** [reply]
i added this to a PR in the PR stack on my fork.
